src/windows/ride.c

```diff
@@ -1096,6 +1096,54 @@ static void window_ride_draw_tab_images(rct_drawpixelinfo *dpi, rct_window *w)
 }
 
 /**
+*
+* rct2: 0x006AEB9F
+*/
+void window_ride_disable_tabs(rct_window *w)
+{
+	//RCT2_CALLPROC_X(0x006AEB9F, 0, 0, 0, 0, (int)w, 0, 0);
+
+	int disabled_tabs = 0;
+	rct_ride *ride = GET_RIDE(w->number & 0xFF);
+
+	int ride_type = ride->type; // ecx
+
+	if ((RCT2_GLOBAL(0x97CF40 + ride_type * 8, uint32) & 0x200) == 0)
+		disabled_tabs |= (1 << WIDX_TAB_8); // 0x800
+
+	if ((ride_type & 0xFF) == RIDE_TYPE_MINI_GOLF)
```

Comment:
I'm being a bit pedantic here but `& 0xFF` is not required as the ride_type is always a byte.